sirepo/package_data/static/js/raydata.js

```diff
@@ -370,10 +381,13 @@ SIREPO.app.directive('scansTable', function() {
         `,
         controller: function(appState, errorService, panelState, raydataService, requestSender, $scope, $interval, $timeout) {
             $scope.analysisModalId = 'sr-analysis-output-' + $scope.analysisStatus;
+            // POSIT: Matches _AnalysisStatus.NONE in scan monitor
+            $scope.analysisStatusNone = "none";
```

Comment:
I don't believe this needs to be on scope. Also, let's move it to a const (UPPER_CAMEL_CASE) inside of raydataService. 

The same can be done for defaultColumns and the ["pending", "running"] on line 507.

Reply:
18d9639913cbfbaea48046856468f9dcbb7714aa